src/Resources/app/administration/src/module/swag-paypal-agent-commerce/extension/sw-sales-channel-detail-base/index.ts

```diff
@@ -0,0 +1,18 @@
+import template from './sw-sales-channel-detail-base.html.twig';
+import { PAYPAL_AGENT_COMMERCE_SALES_CHANNEL_TYPE_ID } from "SwagPayPal/constant/swag-paypal.constant";
+
+export default Shopware.Component.wrapComponentConfig({
+    template,
+
+    computed: {
+        isAgentCommerceType(): boolean {
+            // @ts-expect-error - salesChannel is defined in the parent component
+            // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access,@typescript-eslint/no-unsafe-return
+            return this.salesChannel && this.salesChannel.typeId === PAYPAL_AGENT_COMMERCE_SALES_CHANNEL_TYPE_ID;
```

Comment:
Would this be a bit clearer?
```suggestion
            return this.salesChannel?.typeId === PAYPAL_AGENT_COMMERCE_SALES_CHANNEL_TYPE_ID;
```